style.css

```diff
@@ -0,0 +1,1086 @@
+  align-items: center;
+}
+
+.topNav-left {
+  display: flex;
+  align-self: flex-end;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+  width: 30%;
+  margin-left: 20px;
+}
+
+.topNav-right {
+  display: flex;
+  align-self: flex-end;
+  height: 100%;
+  align-items: center;
+  justify-content: space-between;
+  width: 15%;
+  margin-right: 20px;
+}
+
+.main-nav {
+  width: 100%;
+  display: inline-flex;
+  justify-content: space-between;
+  height: 48px;
+  margin: 0;
+  top: 0;
+}
+
+.main-nav a {
+  color: #aaa;
+}
+
+.nav-logo {
+  width: auto;
+  height: 100%;
+  display: none;
+}
+
+.mainNav-left {
+  align-items: center;
+  width: 47%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.mainNav-right {
+  display: flex;
+  justify-content: center;
+  align-self: flex-end;
+  align-items: center;
+  margin-right: 20px;
+  height: 100%;
+  width: auto;
+}
+
+.mainNav-right i {
+  font-size: 1.5em;
+  color: #aaa;
+}
+
+.header-img {
+  width: 99vw;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  height: 30vw;
+}
+
+.header-img .main-header-img {
+  width: 60%;
+  height: 100%;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+}
+
+.header-img .sub-header-img {
+  width: 39%;
+  height: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: column;
+}
+
+.sub-img-text > div {
+  z-index: 10;
+}
+
+.header-img .sub-header-img > div {
+  width: 100%;
+  height: 49%;
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-clip: content-box;
+  position: relative;
+}
+
+.logo-date {
+  display: flex;
+  width: 100%;
+  height: 30%;
+  justify-content: space-between;
+}
+
+.logo-date .cover-logo {
+  height: 100px;
+  width: 50%;
+  z-index: 2;
+}
+
+.logo-date .article-date {
+  color: white;
+  padding: 15px 15px;
+  z-index: 2;
+}
+
+.cover-text {
+  padding-top: 100px;
+  height: 50%;
+  z-index: 2;
+}
+
+.cover-text h3 {
+  color: white;
+  font-size: 2.4em;
+  font-weight: 800;
+  width: 90%;
+  margin: 0;
+  padding: 0;
+  padding: 0 10px;
+  z-index: 2;
+}
+
+.sub-img-text {
+  margin-top: 15%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  z-index: 2;
+  padding: 10px 15px;
+}
+
+.sub-img-text h3 {
+  color: white;
+  font-weight: 700;
+  font-size: 1.5em;
+  z-index: 2;
+  margin: 0;
+  padding: 0;
+}
+
+.sub-img-text > div {
+  z-index: 10;
+}
+
+.sub-img-text small {
+  z-index: 10;
+  padding-right: 10px;
+}
+
+.sub-img-text time {
+  z-index: 10;
+  color: white;
+}
+
+.sub-img-text > div small {
+  color: white;
+}
+
+.lastest-elements small {
+  font-size: 0.8em;
+  font: inherit;
+  color: #555;
+}
+
+.cover-text .byline small {
+  color: white;
+  padding: 10px;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and use in each HTML tag where you need it.
- [ ] Delete all your `color: white; ` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.color-white{
  color: white;
}
```
**Html file**

```HTML
<div class="color-white">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_